dev-client/datatypes/sites.ts

```diff
@@ -0,0 +1,17 @@
+export type Point2D = [number, number];
+
+export class DisplaySite {
```

Comment:
question: in the web client we haven't been using classes, this type would've been written as
```ts
type DisplaySite = {
  lat: number;
  lon: number;
  name: string;
}
```
if you're indifferent, i think we should also avoid classes in this repo for consistency

Reply:
Agreed, let's keep it consistent.